database/changesets/20201031_WorkflowOptions.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="jwarren@broadinstitute.org"
+               dbms="postgresql"
+               id="20201031"
+               logicalFilePath="20201031_WorkflowOptions.xml">
+        <comment>
+            Add workflow options attributes to existing workflow types
+        </comment>
+        <sql>
+            ALTER TYPE AllOfUsArrays
+                ADD ATTRIBUTE workflow_options   text  CASCADE   -- workflow options to be passed to Cromwell
+        </sql>
+        <sql>
+            ALTER TYPE copyfile
+                ADD ATTRIBUTE workflow_options   text  CASCADE   -- workflow options to be passed to Cromwell
+        </sql>
+        <sql>
+            ALTER TYPE ExternalWholeGenomeReprocessing
+                ADD ATTRIBUTE workflow_options   text  CASCADE   -- workflow options to be passed to Cromwell
+        </sql>
+        <sql>
+            ALTER TYPE CromwellWorkflow
+                ADD ATTRIBUTE workflow_options   text  CASCADE   -- workflow options to be passed to Cromwell
+        </sql>
+        <comment>
+            Add workflow options column to overall workload table
+        </comment>
+        <sql>
+            ALTER TABLE workload ADD workflow_options text
```

Comment:
Sorry I didn't spot this before.
I don't think adding workflow options to the workload table is the best. I'd just merge them with the per-workflow inputs and store them in the workflow table.
I realise that this means we'll be storing duplicate information, but I don't think this information should live in the workload table. 

Maybe in the future, we'd have 3 tables, one for the workload, one for the common features of the workflows, and one for the workflows itself. Who knows.

Reply:
Ah, this was why I wanted to make sure my DB stuff was on the right path before I did everything else.

Why should we not store the workflow options at a per-workload level? If we don't, our API becomes lossy in that we forget about the nature of some options described by the user. As the documentation page describes, the workload-level options returned is also key for a user differentiating between WFL's per-workload defaults and per-workflow defaults.

Reply:
I could also add a new table just to associate options to workloads, if we didn't want to just add a column to the table. I don't know what that gets us with regard to database design but I don't see a reason why we shouldn't store the per-workload options. I suppose it might've lead to a simpler implementation, but the work here is done and tested so I don't know if that's moot.

Reply:
Why do we care "what is common"?  (Maybe we should talk about this.)

Reply:
Users can provide options to be common, and WFL treats some options as being common across a workload (util/make-options, for the most part). Yes, those things end up being visible on the workflows, but they could've been overridden there. Keeping track of what is common allows us to give clear output to the user and also allows this change to work with modules that use an append system, like AoU.
(Here I'm using "common" == "per-workload" == "baseline for every workflow in the workload")

Reply:
OK. But make-options is just implementation.
Letting a user specify options to apply to all workflows is useful.
But that doesn't necessitate reflecting it in the database.
WFL could merge options into workflows at workload creation.